Sources/ApolloCodegenLib/ApolloSchemaDownloader.swift

```diff
@@ -266,9 +266,11 @@ public struct ApolloSchemaDownloader {
     """
   
   
-  static func downloadViaIntrospection(from endpointURL: URL,
-                                       httpMethod: ApolloSchemaDownloadConfiguration.DownloadMethod.HTTPMethod,
-                                       configuration: ApolloSchemaDownloadConfiguration) throws {
+  static func downloadViaIntrospection(
```

Comment:
These are mostly formatting changes right - am I missing anything else?

Reply:
Yupp, just cleaning up the formatting here.